lib/mix/tasks/igniter.install_phoenix.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Mix.Tasks.Igniter.InstallPhoenix do
+  use Igniter.Mix.Task
+
+  @example "mix igniter.install_phoenix"
+  @shortdoc "Install Phoenix project files"
+
+  @moduledoc """
+  #{@shortdoc}
+
+  ## Example
+
+  ```bash
+  #{@example}
+  ```
+
+  ## Options
+
+  # TODO: phx.new options (--umbrella, --no-ecto, etc)
+  # https://github.com/phoenixframework/phoenix/blob/7586cbee9e37afbe0b3cdbd560b9e6aa60d32bf6/installer/lib/mix/tasks/phx.new.ex#L13
+  """
+
+  def info(_argv, _source) do
+    %Igniter.Mix.Task.Info{
+      group: :igniter,
+      example: @example,
+      positional: [:base_path]
+    }
+  end
+
+  def igniter(igniter) do
+    # TODO: check elixir version - https://github.com/phoenixframework/phoenix/blob/7586cbee9e37afbe0b3cdbd560b9e6aa60d32bf6/installer/lib/mix/tasks/phx.new.ex#L380
```

Comment:
`Phx.New` now requires Elixir `~> 1.15` (a commit from 2w ago). Should we impose the same requirement?

Reply:
I'm comfortable with requiring `~> 1.15`, yes.